src/Microsoft.TestPlatform.Common/Resources/Resources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="CancellationRequested" xml:space="preserve">
+    <value>Cancelling the operation as requested.</value>
```

Comment:
Something like "The operation was cancelled." seems better. We can check the existing IDE cancellation message to get more input.

Reply:
Okay. I just took this as reference. https://github.com/Microsoft/vstest/blob/09057425a192fd329c32bbee2fd9bdd401751450/src/Microsoft.TestPlatform.CrossPlatEngine/Resources/Resources.resx#L180 
And I guess we should remove this string.